rules/windows/privilege_escalation_uac_bypass_mock_windir.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/10/26"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via Mocking Microsoft Trusted Windows directory. Attackers bypass UAC to
+stealthily execute code with elevated permissions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "UAC Bypass Attempt via Mocking Windows Directory"
```

I'm not a huge fan of `mocking`. Maybe we use something like mimic, impersonate, imitate, masquerade, etc., for this and the description. Maybe `UAC Bypass Attempt via Windows Directory Masquerading`
```suggestion
name = "UAC Bypass Attempt via Windows Directory Masquerading"
```